### DISCUSSION TITLE
I don't know how to press Ctrl+Alt+Delete on the VM

### DISCUSSION THREAD

**Discussion by goreghost8** [utmapp/UTM#3144] [UTM Help (macOS)] [answered]
Hi, may sound like a really dumb question, but I don't know how to press that key combination and send it to the VM. I´m trying to use Windows Server 2019 but I can't pass the login screen because that. I use a Mac Keyboard.

**Comment by jaigak** [comment, marked as answer]
Press Ctrl+Option+Fn+Delete.
